app/controllers/licence_controller.rb

```diff
@@ -14,101 +14,62 @@ class LicenceController < ApplicationController
   NO_MATCHING_AUTHORITY = 'noLaMatch'.freeze
   NO_MAPIT_MATCH = 'fullPostcodeNoMapitMatch'.freeze
 
-  # NOTE: This is a temporary fix to ensure that these licences get treated as
-  # 'county/unitary' tiered (as opposed to 'district/unitary'). The tier data
-  # for licences used to be stored in LocalService model records in the content
-  # api, but the entries for the licences below have since been removed. In
-  # future this tier data will be stored in the licensing application
-  # (Licensify). Once that has happened and Frontend has been updated to use
-  # that tier information, this list and related code in the
-  # `local_authority_slug` method can be removed
-  LICENCE_SLUGS_WITH_COUNTY_TIER_OVERRIDE = [
-    'scaffolding-and-hoarding-licence',
-    'skip-operator-licence',
-    'permission-to-place-tables-and-chairs-on-the-pavement',
-    'pavement-or-street-display-licence',
-    'petroleum-storage-licence',
-    'weighbridge-operator-certificate',
-    'performing-animals-registration',
-    'approval-of-premises-for-civil-marriage-or-civil-partnership',
-    'licence-projection-over-highway-england-wales',
-  ].freeze
-
   def search
```

Comment:
Quite probably not, as it's only a 'search' for when a licence is local authority specific! That's only true for ~20% of our published LicenceEditions